### PR TITLE
Team PowerDrill - Feat: Contact page working. - ZCG_12

### PR DESCRIPTION
**Pull request by Harrylever**
Contact Page working. Disabled `getuserinfo` function from Contact Form Component, user Email Address entered manually. Modified user.js, not return error upon user not join organisation or workspace yet.

### CODE REVIEW

**Comment by JulianaSau** [comment]
@Harrylever resolve the conflicts on your branch and include an image or video showing how the component looked before and how it looks after your implemented changes

**Comment by Harrylever** [comment]
Just seeing this now would start working on it right away.

**Comment by Harrylever** [comment]
@JulianaSau Just saw this now would start working on it right away.

File: packages/main/src/pages/static/contact-us/components/ContactFormContainer.jsx [hunk 1]
```diff
@@ -31,14 +31,14 @@ function ContactFormContainer() {
...
   });
 
-  useEffect(() => {
-    let userInfo = getUserInfo();
-    setUserAuth(userInfo.user.email ? userInfo.user : {});
-    setValues(values => ({
-      ...values,
-      email: userAuth.email ? userAuth.email : values.email
-    }));
-  }, []);
```
Comment (JulianaSau) [inline packages/main/src/pages/static/contact-us/components/ContactFormContainer.jsx:41]: Please explain why you removed this code and what code you added to substitute this functionality. I'm trying to see the thought process behind this and why this code would not work

**Review by JulianaSau — CHANGES_REQUESTED** [review]
Okay. Also, when you're done, could you provide a video demonstrating how it works now that you've fixed it

**Pull request closed without merging**